src/vma/util/agent.cpp

```diff
@@ -53,6 +53,10 @@
 #undef  MODULE_HDR
 #define MODULE_HDR      MODULE_NAME "%d:%s() "
 
+#define AGENT_DEFAULT_MSG_NUM    (512)
+#define AGENT_DEFAULT_MSG_GROW   (16)
+#define AGENT_DEFAULT_INACTIVE   (10)
+#define AGENT_DEFAULT_ALIVE      (10)
```

Comment:
10 seconds sounds like a long duration. can we reduce it ?

Reply:
we can but I would like to keep it not to small because this scenario should be abnormal and checking frequently can impact normal vma operations.

Reply:
ok